tutorials/observing_examples.ipynb

```diff
@@ -0,0 +1,372 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from synphot import Observation\n",
+    "import numpy as np\n",
+    "\n",
+    "obs = Observation(photlam_hatp11, agile)\n",
+    "obs.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To get the expected count rate for a specific collecting area, use the <a href=\"https://synphot.readthedocs.io/en/latest/api/synphot.observation.Observation.html#synphot.observation.Observation.countrate\">countrate()</a> method: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "11852206.200075638 ct / s\n"
+     ]
+    }
+   ],
+   "source": [
+    "r = 3.5 / 2 * u.m  # radius of 3.5m ARC telescope at APO\n",
+    "area = np.pi * r.to(u.cm) ** 2\n",
+    "\n",
+    "cr = obs.countrate(area=area)\n",
+    "print(cr)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "How many photons can be expected in a 10 second exposure?"
```

Comment:
A more interesting question for an observer (at least what I would want to know back in my observing days) is, "How much exposure time is needed if my target SNR is 10?" (You can replace 10 with what makes sense for your science here.) I think at this point, you have enough info to calculate that?

Reply:
I agree - maybe this is a kind of function we can implement as a part of my project?

Reply:
For Poisson noise-limited photometry, the S/N is just `1/sqrt(counts)`, no? If so, I'm not sure we need to write a special method to do that math.

Reply:
Hmm... `N = sqrt(counts)`, so `counts / sqrt(counts) = sqrt(counts)` is the SNR, and not `1/sqrt(counts)`? That is a naive assumption though. With telescopes, you also need to factor in the sky and readnoise. Up to you on how accurate you want it to be.

Even if a special function is needed, we can prototype it in the notebook first, as mentioned during the tag-up.

Reply:
It may be naive but I'm not sure we can do much better for ground-based observations without sky brightness models, which are complicated (this is something we've come upon in [astroplan again and again](https://github.com/astropy/astroplan/issues/29)).